style.css

```
@@ -0,0 +1,15 @@
+body {
+  margin: 0;
+}
+header {
+  color: white;
+  background-color: rgb(158, 161, 212);
+  height: 100px;
+  display: flex;
+  justify-content: space-around;
```

div에 있는 align을 전체에 적용시켜주면 예시와 더 비슷해질 것 같아요
```suggestion
  justify-content: space-around;
  align-item: center;
```

---

아니면 h1을 div로 감싸주세요!

---

8ef65dca39fe6213f0f16a42adc386911e1166ea
피드백 반영 완료했습니다.